app/retail/views.py

```diff
@@ -132,9 +132,16 @@ def about(request):
         package = (lr.avatar_url, lr.github_username, lr.github_username)
         community_members.append(package)
 
+    alumnis = [
+    ]
+    for alumni in Alumni.objects.filter(public=True).exclude(organization='gitcoinco'):
```

Comment:
Whenever we're accessing related objects, we'll want to use `select_related` or `prefetch_related` to consolidate our queries.  `for alumni in Alumni.objects.select_related('profile').filter(public=True).exclude(organization='gitcoinco'):` will drop the query count on this greatly.